src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * @emails react-core
+ * @flow
+ */
+
+import React, {useState} from 'react';
+import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
+import {sharedStyles} from 'theme';
+
+const FeedbackForm = () => {
+  const [feedbackGiven, setFeedbackGiven] = useState(false);
+
+  if (feedbackGiven) {
+    return 'Thanks for letting us know!';
+  } else {
+    return (
+      <>
+        Is this page useful?
+        <button
+          css={[sharedStyles.articleLayout.feedbackButton, {marginLeft: '6px'}]}
+          aria-label="Yes"
+          onClick={e => {
+            e.preventDefault();
+            trackCustomEvent({
+              category: 'Feedback Button',
+              action: 'feedback',
+              label: window.location.pathname,
+              value: 1,
+            });
```

Comment:
I don't know much about how our GA stuff works. I assumed these buttons are wired up correctly and have been confirmed to work? 😄